HackerRankDashboard/CoreCS/DataStructures/src/main/java/com/javaaid/hackerrank/solutions/datastructures/arrays/DynamicArray.java

```diff
@@ -30,14 +30,14 @@ public DynamicArray(int N) {
 
 	void appendValue(int x, int y, int N) {
 		int rowIndex = (x ^ lastAns) % N;
-		List<Integer> seq = seqList.get(rowIndex);
+		seq = seqList.get(rowIndex);
```

Comment:
If you store the result in existing global variable seq, you will loose the initial input ref list.
So its better to use local variable instead of global.